jooq/src/main/kotlin/com/terraformation/backend/jooq/TerrawareGenerator.kt

```diff
@@ -178,7 +181,7 @@ class TerrawareGenerator : KotlinGenerator() {
         """
       enum class $enumName(
           $properties
-      ) : EnumFromReferenceTable<$idType, $enumName>$localizableExtendsSnippet {
+      ) : EnumFromReferenceTable<$idType, $enumName>$localizableExtendsSnippet$parameterizedExtendSnippet {
```

Comment:
I'm not sure the variable is adding anything here since its value is a constant and we're already referencing an interface of ours directly here anyway. This could be something like
```
) : EnumFromReferenceTable<$idType, $enumName>, ParameterizedEnum$localizableExtendsSnippet
```
Alternately, make `EnumFromReferenceTable` extend the new interface and then it doesn't need to be explicitly listed here at all.